From b067f3698db666899c58869ee1a1946c7003de80 Mon Sep 17 00:00:00 2001
From: James Kosin <jkosin@apache.org>
Date: Tue, 20 Mar 2012 02:01:27 +0000
Subject: [PATCH] OPENNLP-471: found after we find a name match, we don't jump
 over the found name but re-process... thanks William for pointing this out

git-svn-id: https://svn.apache.org/repos/asf/opennlp/trunk@1302751 13f79535-47bb-0310-9956-ffa450edef68
---
 .../main/java/opennlp/tools/namefind/DictionaryNameFinder.java  | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/opennlp-tools/src/main/java/opennlp/tools/namefind/DictionaryNameFinder.java b/opennlp-tools/src/main/java/opennlp/tools/namefind/DictionaryNameFinder.java
index b804ddc6d..e2b3754ea 100644
--- a/opennlp-tools/src/main/java/opennlp/tools/namefind/DictionaryNameFinder.java
+++ b/opennlp-tools/src/main/java/opennlp/tools/namefind/DictionaryNameFinder.java
@@ -71,6 +71,8 @@ public Span[] find(String[] tokenStrings) {
 
       if (foundName != null) {
         foundNames.add(foundName);
+        /* skip over the found tokens for the next search */
+        startToken = (foundName.getEnd() - 1);
       }
     }
 
